### PR TITLE
SAK-40209 add category statistics popup to GBNG

### PR DESCRIPTION
**Pull request by RyanAFinney**
Added Category statistics popup in a dropdown menu on category columns

### CODE REVIEW

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/GradebookPage.java [hunk 1]
```diff
@@ -104,6 +87,7 @@
 	GbModalWindow gradeCommentWindow;
 	GbModalWindow deleteItemWindow;
 	GbModalWindow assignmentStatisticsWindow;
+	GbModalWindow categoryGradeStatisticsWindow;
```
[the remark this answers is not in the document]
Comment (RyanAFinney) [reply gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/GradebookPage.java]: Leaving for consistency.

**Comment by steveswinsburg** [comment]
This is great, but I'm going to put a hold on it for the time being. I am making an update to the stats panels so that they can be shared across assignment/coursegrade and now categories. So bear with me then you should be able to throw a DescriptiveStatistics object at it and it will render.

**Comment by RyanAFinney** [comment]
Sounds good. I'll keep an eye out for it-- thanks!

**Comment by steveswinsburg** [comment]
#5736 contains the framework for this. Extend `org.sakaiproject.gradebook.tool.stats.BaseStatistics` and drop it into a modalwindow (like the `AssignmentStatisticsPanel` does). You can add additional markup to the panel as well (see `CourseGradeStatistics`). Let me know if you have any questions. Cheers.

**Pull request closed without merging**